### PR TITLE
Update Lisk SDK to v6.0.0-beta.0 

### LINKED ISSUE #105: Update Lisk SDK to v6.0.0-beta.0
### Description

Update Lisk SDK to v6.0.0-beta.0

### Acceptance Criteria

- Update Lisk SDK to v6.0.0-beta.0
- Update the banner image

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?

This PR resolves #105 

### How was it solved?
- [x] Update Lisk SDK to v6.0.0-beta.0
- [x] Update the banner image

### How was it tested?
Local against `testnet`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,19 +61,19 @@
 	"dependencies": {
 		"@liskhq/lisk-api-client": "5.1.6",
 		"@liskhq/lisk-chain": "0.3.4",
```
Comment (priojeetpriyom) [inline package.json:63]: Don't we need to update this to beta?

@liskhq/lisk-api-client@6.0.0-beta.0
@liskhq/lisk-chain@0.4.0-beta.0

File: package.json [hunk 2]
```diff
@@ -61,19 +61,19 @@
...
-		"@liskhq/lisk-codec": "0.3.0-alpha.14",
+		"@liskhq/lisk-codec": "0.3.0-beta.0",
 		"@liskhq/lisk-cryptography": "3.2.1",
 		"@liskhq/lisk-db": "0.2.1",
 		"@liskhq/lisk-utils": "0.2.1",
```
Comment (priojeetpriyom) [inline package.json:67]: @liskhq/lisk-utils@0.3.0-beta.0?

File: package.json (hunk 1 above)
Comment (sameersubudhi) [reply package.json:63]: This is correct. We need the older versions to connect to Lisk Core v3+.

File: package.json [hunk 3]
```diff
@@ -61,19 +61,19 @@
 	"dependencies": {
 		"@liskhq/lisk-api-client": "5.1.6",
 		"@liskhq/lisk-chain": "0.3.4",
-		"@liskhq/lisk-codec": "0.3.0-alpha.14",
+		"@liskhq/lisk-codec": "0.3.0-beta.0",
 		"@liskhq/lisk-cryptography": "3.2.1",
```
[the remark this answers is not in the document]
Comment (sameersubudhi) [reply package.json:65]: Can be left as is.

**Review by sameersubudhi — APPROVED** [review]

**Pull request merged**